backend/tests/unit/llm/providers/test_anthropic.py

```
@@ -410,3 +410,207 @@ def test_parse_response_empty_content_with_tool_calls(self):
+        # Arrange: プロバイダーとメッセージを準備
+        provider = AnthropicProvider("test-key", "claude-3-5-sonnet-20241022")
+        messages = [Message(role="user", content="Get my stats")]
+
+        # ストリーミングレスポンスをモック
+        mock_response = MagicMock()
+        mock_response.status_code = 200
+        mock_response.raise_for_status = MagicMock()
+
+        async def mock_iter_lines():
+            lines = [
+                # 完了イベント（ツール使用の場合、stop_reasonがtool_useになる）
```

Ruff (0.14.13)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)